drivers/at86rf2xx/at86rf2xx_internal.c

```diff
@@ -211,6 +211,25 @@ void at86rf2xx_configure_phy(at86rf2xx_t *dev)
     at86rf2xx_set_state(dev, state);
 }
 
+#if defined MODULE_AT86RF233 || defined MODULE_AT86RF231
```

Comment:
could you write those macro calls function like? `defined(MODULE_…)`. It looks a little bit more concise in my opinion.